images/linux/scripts/installers/google-cloud-sdk.sh

```diff
@@ -10,7 +10,7 @@ REPO_URL="https://packages.cloud.google.com/apt"
 echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] $REPO_URL cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
 curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
 sudo apt-get update -y
-sudo apt-get install -y google-cloud-sdk
+sudo apt-get install -y google-cloud-sdk=369.0.0-0
```

Comment:
let's add a comment why do we need to hardcode the version

Reply:
@miketimofeev done